### DISCUSSION TITLE
NO CURRENT RESULT - API keys don't work?

### DISCUSSION THREAD

**Discussion by adamcurrey** [cytech/Home-Assistant-wundergroundpws#143] [Q&A]
Hi,

Possibly silly question - does the WU API still work?  I know it got shut down a couple of years ago, and reinstated, but all I get is "NO CURRENT RESULT".  I have a PWS currently reporting to WU.

**Comment by cytech** [comment]
closed as outdated

**Comment by cytech** [comment]
closed as outdated
